packages/ai/src/authorizers/federated-connections/index.ts

```diff
@@ -7,12 +7,14 @@ export type { AsyncStorageValue } from "./asyncLocalStorage";
 
 export { asyncLocalStorage };
 
-export type FederatedConnectionAuthorizerParams<ToolExecuteArgs extends any[]> =
-  {
-    refreshToken: AuthorizerToolParameter<ToolExecuteArgs, string>;
-    scopes: AuthorizerToolParameter<ToolExecuteArgs, string[]>;
-    connection: AuthorizerToolParameter<ToolExecuteArgs, string>;
-  };
+export type FederatedConnectionAuthorizerParams<ToolExecuteArgs extends any[]> = {
+  refreshToken?: AuthorizerToolParameter<ToolExecuteArgs, string>;
```

Comment:
I would also rename this to getRefreshToken to keep it consistent or rename the following param to `accessToken`